polyphemus/lib/commands.rb

```diff
@@ -13,17 +13,129 @@ def execute
     end
   end
 
-  class GetTree < Etna::Command
-    usage 'Open a console with a connected Polyphemus instance.'
+  module WithEtnaClients
+    def project
+      raise "project must be implemented in subclasses!"
+    end
+
+    def token
+      Polyphemus.instance.config(project)[:token]
+    end
+
+    def magma_client
```

Comment:
@coleshaw I moved the magma_client logic out of the application object into here, since these clients will have per project tokens.